### ISSUE TITLE
Support diffing APIs

### ISSUE THREAD

**Issue by MattouFP** [filefog/filefog#7]
Several Cloud providers support getting a list of files that changed between the last check and the request time.

https://developers.google.com/drive/web/manage-changes
https://dev.onedrive.com/items/view_changes.htm
https://www.dropbox.com/developers/core/docs#delta

**Comment by AnalogJ** [comment]
Hey, I'm definitely open to that. It would probably have to be a custom interface like `events` and `webhooks`. If you want to create a PR I'll look at merging it.

**Comment by MattouFP** [comment]
Or maybe just a new method like getChangesSince(changeToken) that returns a list of changed elements between the time the token was redeemed and now?

**Comment by AnalogJ** [comment]
yeah, that sounds good to me. It would still need to be a custom interface though, because its not a standard function that is available on all providers. I want the default interface to just be standard file and folder CRUD operations basically.
